overlays/haskell.nix

```diff
@@ -514,6 +514,7 @@ final: prev: {
                   tool = final.buildPackages.haskell-nix.tool pkg-set.config.compiler.nix-name;
                   tools = final.buildPackages.haskell-nix.tools pkg-set.config.compiler.nix-name;
                   roots = final.haskell-nix.roots pkg-set.config.compiler.nix-name;
+                  projectFunction = haskell-nix: haskell-nix.cabalProject' args';
```

Comment:
I don't love this... what if we instead just passed `args'` so it could re-call `cabalProject'` with the right args if it needs to?

Reply:
I included `projectFunction` because I planned to add it for `stackProject'` as well, but forgot to update that (it is added now).  I have included `projectArgs` for the args so that `projectFunction` does as little as possible.